src/components/RequestToBeProvider.svelte

```diff
@@ -89,7 +147,47 @@
   const clearTxnStatuses = () => (txnStatuses = new Array<string>());
 </script>
 
-<div id="request-to-be-provider" class="action-card basis-1/2">
+  <div id="request-to-be-provider" class="content-block w-single-block flex flex-col gap-4">
+    <BlockSection title="Become a provider">
+      <DropDownMenu
+        id="network"
+        label="Select a Network"
+        bind:action={selectedNetworkAsString}
+        onChange={networkChanged}
+        placeholder=""
+        options={networks}
+      ></DropDownMenu>
+      <input
+        type="text"
+        id="other-endpoint-url"
+        placeholder="wss://some.frequency.node"
+        bind:value={customNetwork}
+        disabled={$selectedNetwork != Network.CUSTOM}
+        class:hidden={$selectedNetwork != Network.CUSTOM}
```

Comment:
hidden or if comment. up to you.

Reply:
If you select Custom, there is an infinite error.
<img width="580" alt="Screenshot 2024-01-23 at 2 30 49 PM" src="https://github.com/LibertyDSNP/provider-dashboard/assets/43625033/9856fc52-d812-4d01-87d4-f02735941451">